### ISSUE TITLE
Already set config.yml but failed

### ISSUE THREAD

**Issue by ikkosatrio** [fspringveldt/ss-easy-s3#7]
i am already set config.yml but when i upload an file i got the error Configuration not found in any yaml file. Please check the ss-easy-s3 README.md file for how to add it.

**Comment by fspringveldt** [comment]
Hi @ikkosatrio 

Can you perhaps paste a copy of your config here? Remove any sensitive info first of course.

**Comment by ikkosatrio** [comment]
oh am already done for it, and now i got error on " <**br />
<b>Fatal error</b>:  Class 'Aws\S3\S3Client' not found in <b>/Applications/XAMPP/xamppfiles/htdocs/silvanus/ss-easy-s3/code/classes/S3Facade.php</b> on line <b>55</b><br />** "

**Comment by fspringveldt** [comment]
That sounds like the AWS SDK did not install properly. 
Did you install this module via composer? It has a requirement on `"aws/aws-sdk-php": "3.18"`, the installation of which composer should handle for you.

**Comment by ikkosatrio** [comment]
yes i install it via composer, maybe autoload composer not called on my Silverstripe , how to call it? or is there any config?

**Comment by fspringveldt** [comment]
Do me a favour, run `composer show` from the root of your project and paste the result here.

Also, silly question, but did you run `dev/build flush=1` after installing this module?

**Comment by ikkosatrio** [comment]
![image](https://user-images.githubusercontent.com/17069028/48326655-169cd200-e66d-11e8-9ae6-6178a7ae9cf2.png)

**Comment by ikkosatrio** [comment]
> Do me a favour, run `composer show` from the root of your project and paste the result here.
> 
> Also, silly question, but did you run `dev/build flush=1` after installing this module?

i am already run "dev/build/?flush=1" too hehe

**Comment by fspringveldt** [comment]
Odd indeed. I'll try and see if I can replicate this and give you feedback.

**Comment by ikkosatrio** [comment]
or my ss doesn't load the composer well?, are you know how to check if our composer is ok or already load by the ss well

**Comment by ikkosatrio** [comment]
hello, i am already fix it, it because my php version wkwkwk.

**Comment by ikkosatrio** [comment]
what i choose RTMP or WEB for upload an files like .docx, .pdf. mp4?

**Comment by fspringveldt** [comment]
Are you following the exact steps in the readme or is your config slighty
different?
